### ISSUE TITLE
Not working with react 0.7.1

### ISSUE THREAD

**Issue by crysfel** [appintheair/react-native-looped-carousel#5]
I have updated my app to the latest version of react native (0.7.1) and the carousel stopped working. I'm getting the red screen with the following error:

```
'Error: \n stack: \n  
<unknown>  index.ios.bundle:22949\n 
 require    index.ios.bundle:254\n 
 <unknown>  index.ios.bundle:22003\n  
require    index.ios.bundle:254\n 
 <unknown>  index.ios.bundle:1480\n  
require    index.ios.bundle:254\n  
<unknown>  index.ios.bundle:1438\n  
require    index.ios.bundle:254\n  
require    index.ios.bundle:200\n 
URL: http://localhost:8081/index.ios.bundle\n line: 22949\n 
message: undefined is not an object (evaluating \'RCTScrollViewConsts.KeyboardDismissMode.None\')'
```

Any thoughts?

**Comment by piyushchauhan2011** [comment]
:+1: Not working for me too.

**Comment by dvdhsu** [comment]
I've fixed this in dvdhsu/react-native-looped-carousel. PR here: https://github.com/appintheair/react-native-looped-carousel/pull/6

**Comment by piyushchauhan2011** [comment]
@dvdhsu Please send the pull request to this master branch and release a new tag or update on npmjs website.

**Comment by srounce** [comment]
Fixed for good in #7

**Comment by spronin** [comment]
Now works with RN 0.11.2
